cms/templates/certificates.html

```diff
@@ -30,8 +32,16 @@
 
 <%block name="requirejs">
   require(["js/certificates/factories/certificates_page_factory"], function(CertificatesPageFactory) {
-      if(${escape_json_dumps(has_certificate_modes)}) {
-          CertificatesPageFactory(${escape_json_dumps(certificates) | n}, "${certificate_url}", "${course_outline_url}", ${escape_json_dumps(course_modes) | n}, ${escape_json_dumps(certificate_web_view_url) | n}, ${escape_json_dumps(is_active) | n}, ${escape_json_dumps(certificate_activation_handler_url) | n} );
+      if(${has_certificate_modes | n, dump_js_escaped_json}) {
+          CertificatesPageFactory(
+              ${certificates | n, dump_js_escaped_json},
+              "${certificate_url | n, js_escaped_string}",
+              "${course_outline_url | n, js_escaped_string}",
+              ${course_modes | n, dump_js_escaped_json},
+              "${certificate_web_view_url | n, js_escaped_string}",
```

Comment:
@nedbat I found a little issue with this update and need your and @cahrens's help in determining what we want to do here.

Right now, if certificate_web_view_url is None, we get "None" on the other end.  If I leave this as `dump_js_escaped_json` without the quotes, you either get back the string or you get `null` with no quotes.

In other places (see below), people do something like the following:
```
courselikeHomeUrl = "${courselike_home_url or "" | n, js_escaped_string}",
```
This will clearly return `""`, rather than `null`.

If `js_escaped_string` is passed None, we have the option of returning `"None"`, `""`, or `null`.
I would propose that we return `""` to be consistent with what most people are using, and to avoid returning null when we can have a valid string instead.

It means that if someone was already using `dump_js_escaped_json`, we can't just blindly switch to `js_escaped_string` because it would change the contract.  So, for these variables that were previously dumping strings as json, even though that isn't our proposed best practice, I will leave them as they were.

Thoughts?